### PR TITLE
Feature/events load

### PR DESCRIPTION
**Pull request by jindazhao**
Confirm Sign Out Button and implementation works.

Created new strings for confirmation of signing out.

### CODE REVIEW

**Comment by jindazhao** [comment]
It builds fine locally I'm not sure why it's failing one of the tests

**Comment by adindotdev** [comment]
I think you're missing a file, did you `git add` everything?

File: lib/constants/strings.dart [hunk 1]
```diff
@@ -42,6 +42,11 @@ class S {
   static const String profileIsNotMember = 'Non-member';
   static const String profileMemberInfoTitle = 'Your Details';
   static const String profileEditDetails = 'Edit details';
+  static const String confirmSignOutTitle = 'Confirm Sign Out?';
```
Comment (adindotdev) [inline lib/constants/strings.dart:45]: Change copy.

File: lib/profile/profile.dart [hunk 2]
```diff
@@ -39,7 +41,9 @@ class _ProfileState extends ProfileView {
                 ),
               ),
             ),
-            ProfileSignOut(onSignOutPressed: widget.presenter.signOutUser),
+            //ProfileSignOut(onSignOutPressed: widget.presenter.onSignOutButtonPressed),
+            ProfileSignOut(onSignOutPressed: widget.presenter.onSignOutButtonPressed),
+            // ProfileSignOutDialog(onConfirmedPressed: widget.presenter.onConfirmButtonPressed),
```
Comment (adindotdev) [inline lib/profile/profile.dart:46]: Remove commented lines.

File: lib/profile/profile_presenter.dart [hunk 3]
```diff
@@ -65,11 +67,20 @@ class ProfilePresenter extends Presenter<ProfileView, ProfileModel> {
...
           MaterialPageRoute(builder: (context) => Login(LoginPresenter())));
     }
   }
+
+  void onSignOutButtonPressed(BuildContext context) {
+    showDialog(
+      context: context,
+      barrierDismissible: false,
+      builder: (context) =>
+          ProfileSignOutDialog(onConfirmedPressed: onConfirmButtonPressed),
```
Comment (adindotdev) [inline lib/profile/profile_presenter.dart:83]: File including ProfileSignOutDialog widget is missing.

**Review by adindotdev — CHANGES_REQUESTED** [review]

**Comment by jindazhao** [comment]
I pushed with the new updates but still unsure as to why the build is failing

**Pull request closed without merging**